src/utils/utils.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Returns formatted date eg Wed Jul 28 1993.
+ *
+ * @param {date} dateTime
+ * @returns {date}
+ */
+export function dateFormatter(dateTime) {
+  const date = new Date(dateTime);
+
+  return date.toDateString();
+}
+
+/**
+ * Fetch and return responses from Github apis.
+ *
+ * @param {*} url
+ * @param {*} GITHUB_TOKEN
+ * @returns {Promise}
+ */
+export function fetchInfoFromGitHub(url, GITHUB_TOKEN) {
```

Comment:
variables must be in camelCase.
```suggestion
export function fetchInfoFromGitHub(url, githubToken) {
```